src-script/pack-apack-mac.js

```diff
@@ -5,31 +5,48 @@ const sevenBin = require('7zip-bin')
 const pathTo7zip = sevenBin.path7za
 
 exports.default = async function (buildResult) {
-  buildResult?.artifactPaths?.forEach((element) => {
+  for (const element of buildResult?.artifactPaths || []) {
```

Comment:
Why is this needed?

Reply:
because .forEach() does not await Promises inside its callback